src/dug/core/tranql.py

```diff
@@ -142,7 +142,11 @@ def get_kg(self):
         for q_id in query_graph["nodes"]:
             node_details = query_graph["nodes"][q_id]
             node_curie = node_details.get("id", "")
-            node_type = [self._snake_case(x.replace('biolink:', '')) for x in node_details.get("category", [])]
+            category = node_details.get("category", [])
+            if type(category) == list:
+                node_type = [self._snake_case(x.replace('biolink.', '')) for x in category]
```

Comment:
can we store the `biolink.` in the config object somewhere and pass it down ? 
I think that way we could use this code with the current graph, and later change the config and would be able to run it on multiple labeled graph aswell ?